components/scream/tests/coupled/dynamics_physics/model_restart/input_baseline.yaml

```diff
@@ -14,9 +14,12 @@ Time Stepping:
 Initial Conditions:
   Filename: ${SCREAM_DATA_DIR}/init/screami_ne2np4L72_20220524.nc
   Restart Run: false
-  Physics GLL:
-    surf_evap: 0.0
-    surf_sens_flux: 0.0
+  surf_evap: 0.0
```

Comment:
I'm puzzled why we need quite a few "init to 0" lines in the XML defaults for v1, while here we only need a handful.

Reply:
I think because these aren't using the stripped down input files.